src/Commands/AutoCenterGear.cpp

```diff
@@ -21,16 +21,16 @@ AutoCenterGear::AutoCenterGear() {
 	// e.g. AddParallel(new Command1());
 	//      AddSequential(new Command2());
 	// Command1 and Command2 will run in parallel.
-  AddSequential(new SetAutoShifter(Chassis::kAutoShiftOff));
-  AddSequential(new SetShifter(Chassis::kShifterLow));
-  AddSequential(new AutoDrive(-5.771, 5.567));
+  //AddSequential(new SetAutoShifter(Chassis::kAutoShiftOff));
+  //AddSequential(new SetShifter(Chassis::kShifterLow));
```

Comment:
Normally commented out code should just be deleted. However, I want to debug auto-shifting in order to work out how to make it better.

No change required.